public/locales/en/main.json

```diff
@@ -1,6 +1,7 @@
 {
   "affiliate-unit": {
-    "big-unit-heading": "Suggested For You"
+    "big-unit-heading": "Suggested For You",
+    "disclaimer": "Disclosure: Some of the links above are affiliate links meaning, at no additional cost to you, Fandom will earn a commission if you click through and make a purchase."
```

Comment:
Noticed that the copy in the ticket didn't have a period. Was that a typo in the ticket?

> Disclosure: Some of the links above are affiliate links meaning, at no additional cost to you, Fandom will earn a commission if you click through and make a purchase

Reply:
I added the period, but I'll remove it just in case.